src/shared/components/cells/CellContext.tsx

```diff
@@ -80,21 +83,6 @@ const CellContext: FC<Props> = ({
     event('editCell button Click')
   }
 
-  const refreshCell = (): void => {
```

Comment:
I think when I originally went through this I was confused by the pause functionality here. I think we still want this in, but we’d want to do a couple things:

1. clean this up by basically using the same function that’s in the Cell.tsx file
2. Make sure you’re passing in the variables as part of the process to reset the query cache